web_console/odc-about-web-terminal.adoc

```diff
@@ -0,0 +1,19 @@
+[id="odc-about-web-terminal"]
+= About the web terminal in the web console
+include::modules/common-attributes.adoc[]
+:context: odc-about-web-terminal
+
+toc::[]
+
+You can launch an embedded command line terminal instance in the OpenShift web console.
+
+<Confirm with Gaja and Abhi on cli tools as well as project context>This terminal instance is pre-installed with common CLI tools for interacting with the cluster, such as `oc`, `kubectl`,`odo`, `kn`, `tkn`, `helm`, `kubens`, and `kubectx`.
```

Comment:
Web terminal comes with these all CLI tools pre-installed and user doesn't need to install anything.

Reply:
@gajanan-more @abhinandan13jan can you confirm that the terminal instance has the context of the project a user is working in too?

Reply:
No. User needs to select the project. By default, `openshift-terminal` project is get selected.

Reply:
User needs to select the project where DevWorkspace CR will be created, by default the current is selected:
![Screenshot_20210111_114252](https://user-images.githubusercontent.com/5887312/104165610-450bc880-5402-11eb-8373-1c7bab3ac5bd.png)

Then when user any
![Screenshot_20210111_114302](https://user-images.githubusercontent.com/5887312/104165872-a0d65180-5402-11eb-9364-05dea99f97ed.png)

it's used as current in kubeconfig and is reset each time after terminal is reopened:
![Screenshot_20210111_114414](https://user-images.githubusercontent.com/5887312/104165924-adf34080-5402-11eb-883c-5cee5c982e18.png)

I remember we talked about setting the current namespace into kubeconfig, but as far as I see it's not implemented yet

For Cluster Admins, there is no choice for target namespace. 
`openshift-terminal` will be always used. More see https://www.youtube.com/watch?v=v6Xy7LKbYrU from https://github.com/openshift/console/pull/7145

Reply:
I think we should explicitly mention: `Cluster admins has terminal access only starting from OpenShift 4.7` somewhere. Since all of that code is on the OpenShift console side, basically a user just needs the newest dashboard and they will get that feature regardless of what version their operator is

Reply:
I will create a separate PR only for 4.7 to add the note on Cluster Admin.